test/e2e/specs/wp-stats-insights-spec.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * External dependencies
+ */
+import config from 'config';
+
+/**
+ * Internal dependencies
+ */
+import LoginFlow from '../lib/flows/login-flow.js';
+
+import NavBarComponent from '../lib/components/nav-bar-component.js';
+
+import StatsPage from '../lib/pages/stats-page.js';
+
+import * as driverManager from '../lib/driver-manager.js';
+
+const mochaTimeOut = config.get( 'mochaTimeoutMS' );
+const startBrowserTimeoutMS = config.get( 'startBrowserTimeoutMS' );
+const screenSize = driverManager.currentScreenSize();
+
+let driver;
+
+before( async function() {
+	this.timeout( startBrowserTimeoutMS );
+	driver = await driverManager.startBrowser();
+} );
+
+describe( 'Stats: (' + screenSize + ') @parallel', function() {
+	this.timeout( mochaTimeOut );
+	describe( 'Log in as user', function() {
+		step( 'Can log in as user', async function() {
+			this.loginFlow = new LoginFlow( driver );
+			return await this.loginFlow.login( { useFreshLogin: true } );
```

Comment:
I would skip using `useFreshLogin` here unless you think otherwise? The test will execute faster using login cookie instead of physically logging in.

Reply:
Sounds good! 👍